.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- Replace <`issueNumber`> with the Issue number to link it with this PR -->
+<!-- Example: #1 links this PR to the first Issue-->
+resolves #`issueNumber`
+
+## Checklist
+
+- [ ] You've followed the code structure 
+- [ ] You've linked the PR to correct Issue
+
+## Proposed Changes
+
+  -
+  -
+  -
```

Comment:
Also add one more field called additional information where one can add more info on the changes like screenshots, dependencies added/removed, major changes to UI/UX, any breaking changes

Reply:
cool! lemme see